### DISCUSSION TITLE
Bug: old/incorrect content rendered in main space view, but correctly rendered in change requests

### DISCUSSION THREAD

**Discussion by liliakai** [GitbookIO/community#699] [0. General bugs and questions]
I have a page in a space (not the main content) which seems to rendered incorrectly given the current markdown I see on github. When I open a change request, the content is re-rendered and correctly reflecting the markdown. I even merged a change request with no lines changed, but still don't get updated content from the main space view. Please fix? Seems like maybe a local frontend caching issue. And in the meantime, is there any way to force the space to render the content correctly?

https://app.gitbook.com/o/WQpOq5ZFue4N6m65QCJq/s/f4g5Jn4MhpjSS8ZADdcN/categories

**Comment by aikoii** [comment]
@liliakai I'm sorry you are facing this issue! We’re looking into it 🙏
